models/fungible/chains/fungible_arbitrum_transfers.sql

```diff
@@ -0,0 +1,16 @@
+{{ config(
+        tags = ['dunesql'],
+        schema = 'fungible_arbitrum',
+        alias=alias('transfers'),
+        file_format = 'delta',
```

Comment:
you don't need this property on a view

Reply:
ah accidentally included it, just removed it